docs/self-managed/platform-deployment/helm-kubernetes/guides/connecting-to-entra-id.md

```diff
@@ -0,0 +1,45 @@
+---
+id: connecting-to-entra-id
+title: "Connecting to Entra ID (Azure AD)"
+description: "Learn how to connect your Camunda Self-Managed deployment to Entra ID (Azure AD)."
+---
+
+:::note
+The following components support Entra ID authentication: Operate
```

Comment:
After reading through this, I get the assumption that additional components will eventually support this? If not, may need to reword a few notes.

Reply:
Thats correct, in the next iterations more components will be added :)